apps/services/sessions/src/app/sessions/create-session.dto.ts

```diff
@@ -41,4 +41,12 @@ export class CreateSessionDto {
   @IsString()
   @ApiProperty()
   ip!: string
+
+  @IsString()
+  @IsOptional()
+  @ApiPropertyOptional({
+    description:
+      'ISO 3166-1 alpha-2 country code: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2',
+  })
+  ipLocation?: string
```

Comment:
Consider implementing a verification to prevent bogus country codes
```suggestion
  @IsIsoCountryCode
  ipLocation?: string
```

Reply:
great idea, done.